### PR TITLE
Configure tags field

### PR DESCRIPTION
**Pull request by ryan953**
Depends on https://github.com/getsentry/sentry-javascript/pull/12197
Follows instructions from https://github.com/getsentry/sentry-docs/pull/10137

### CODE REVIEW

File: sentry.client.config.ts [hunk 1]
```diff
@@ -33,14 +33,18 @@ Sentry.init({
...
-    //     submitBorder: "rgb (0,0,0)",
-    //   },
-    // }),
+    Sentry.feedbackIntegration({
+      colorScheme: "light",
+      showScreenshot: true,
+      themeLight: {
+        submitBackground: "rgb(235,20,20)",
+        submitBackgroundHover: "rgb(140,0,0)",
+        submitBorder: "rgb (0,0,0)",
```
Comment (ryan953) [inline sentry.client.config.ts]:
```suggestion
        accentForeground: "#ffffff",
        accentBackground: "rgb(235,20,20)",
```

File: sentry.client.config.ts [hunk 2]
```diff
@@ -37,10 +37,12 @@ Sentry.init({
...
-        submitBackground: "rgb(235,20,20)",
-        submitBackgroundHover: "rgb(140,0,0)",
-        submitBorder: "rgb (0,0,0)",
-      },
+        accentForeground: "#ffffff",
+        accentBackground: "rgb(235,20,20)",
+      tags: {
+        appName: 'pokemon-market',
+        component: 'autoInjected'
+      }
```
Comment (ryan953) [inline sentry.client.config.ts]:
```suggestion
        accentForeground: "#ffffff",
        accentBackground: "rgb(235,20,20)",
      },
      tags: {
        appName: 'pokemon-market',
        component: 'autoInjected'
      }
```

File: src/components/useFeedback.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import type {RefObject} from 'react';
+import {useEffect} from 'react';
+import * as Sentry from '@sentry/browser';
+import type { OverrideFeedbackConfiguration } from '@sentry/types';
```
Comment (ryan953) [inline src/components/useFeedback.tsx]: This isn't exported anymore (per https://github.com/getsentry/sentry-javascript/pull/12197)

So we'll need some kind of change.

File: pages/feedback.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+        attachments: [],
+      });
+      event.preventDefault();
+    }}>
+      <input name="name" placeholder="Your Name" />
+      <input name="email" placeholder="Your Email" />
+      <textarea name="message" placeholder="What's the issue?" />
+      <input type="file" name="attachment" />
+      <button type="submit" className="hover:bg-hover px-4 py-2 rounded-md">Submit</button>
+    </form>
```
Comment (c298lee) [inline pages/feedback.tsx]:
```suggestion
    <form id="my-feedback-form" onSubmit={async event => {
      const formData = new FormData(event.currentTarget);
      
      const attachment = async () => {
        const data = new Uint8Array(
          await formData.get("attachment").arrayBuffer()
        );
        const attachmentData = {
          data,
          ...formData.get("attachment"),
        };
        return attachmentData;
      };
      
      Sentry.getCurrentScope().setTags({component: 'MyFeedbackForm'});
      Sentry.captureFeedback({
        name: formData.get('name'),
        email: formData.get('email'),
        message: formData.get('message'),
      }, {
        attachments: [await attachment()],
      });
      event.preventDefault();
    }}>
      <input name="name" placeholder="Your Name" />
      <input name="email" placeholder="Your Email" />
      <textarea name="message" placeholder="What's the issue?" />
      <input type="file" name="attachment" />
      <button type="submit">Submit</button>
    </form>
```
```

File: pages/feedback.tsx [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+      Sentry.getCurrentScope().setTags({component: 'MyFeedbackForm'});
+      Sentry.captureFeedback({
+        name: formData.get('name'),
+        email: formData.get('email'),
+        message: formData.get('message'),
+      }, {
+        attachments: [await attachment()],
+      });
+      event.preventDefault();
+    }}>
```
Comment (ryan953) [inline pages/feedback.tsx]:
```suggestion
    <form id="my-feedback-form" onSubmit={async event => {
      event.preventDefault();
      const formData = new FormData(event.currentTarget);
      
      const attachment = async () => {
        const data = new Uint8Array(
          await formData.get("attachment").arrayBuffer()
        );
        const attachmentData = {
          data,
          ...formData.get("attachment"),
        };
        return attachmentData;
      };
      
      Sentry.getCurrentScope().setTags({component: 'MyFeedbackForm'});
      Sentry.captureFeedback({
        name: formData.get('name'),
        email: formData.get('email'),
        message: formData.get('message'),
      }, {
        attachments: [await attachment()],
      });
    }}>
```

**Pull request merged**